web/src/tomviz-viewer.js

```diff
@@ -134,19 +134,18 @@ export function load(container, zipFile, options) {
       createUI(viewer, container);
     });
   });
-  rootQueryDataModel.useZipContent(zipFile, options).then((dm) => {
-    rootQueryDataModel.fetchData();
-  });
+  rootQueryDataModel.useHtmlContent();
+  rootQueryDataModel.fetchData();
 }
 
 // Be ready for file drop -----------------------------------------------------
 const container = document.querySelector('.react-content');
 const fileSelector = document.querySelector('.fileSelector');
 
 const bodyStyle = document.querySelector('body').style;
-bodyStyle.background = `url(${background})`;
-bodyStyle.backgroundRepeat = 'no-repeat';
-bodyStyle.backgroundPosition = '50% 50%';
+// bodyStyle.background = `url(${background})`;
+// bodyStyle.backgroundRepeat = 'no-repeat';
+// bodyStyle.backgroundPosition = '50% 50%';
```

Comment:
It depend if we want to provide a background image at startup or not. 

And we may change our mind later on. But I guess we can always look at the history to find out how to do it again...